pom.xml

```diff
@@ -104,6 +104,14 @@
             <name>JCenter Repository</name>
             <url>https://jcenter.bintray.com/</url>
         </repository>
+         <repository>
```

Comment:
this has already been added by https://github.com/openhab/openhab-distro/pull/2, so please rebase!